users/templates/mail/welcome_and_next_steps.html

```diff
@@ -1,75 +1,56 @@
-            <h1>Welcome {{user.first_name}} to {{ settings.SITENAME }}</h1>
-
-            <p>
-              Great to have you onboard! Your membership application has been
-              approved and now is a great time to to join our community by
-              coming to the next community meeting on the first Tuesday of the
-              month or poping by weekly open house every Tuesday. It is also
-              highly recommendable to join our communication channels (Matrix,
-              Telegram or IRC and email list). Be happy to ask if you like to
-              know more!
-            </p>
-
-            <h2>Payment info</h2>
-            <p>
-              Your personal bank reference number is
-              <b>{{user.reference_number}}</b>. Remember always to use your
-              reference number.
-            </p>
-            <p>
-              We will be waiting your first payment before we add your account
-              access in to the systems. Before this you can join our community
-              meetings, open house and communication channels and ask if someone
-              could help you. All Hacklab services work by voluntary basis so
-              things might not work always. Please be patient. For the
-              challenges help would be appreciated, and we would be more than
-              happy for your contribution.
-            </p>
-
-            <h3>Your services</h3>
-
-            <ul>
-            {% for servicesubscription in user.servicesubscription_set.all %}
-                <li>{{ servicesubscription.service.name }} - {{ servicesubscription.service.cost_string }} / {{ servicesubscription.service.period_string }} days</li>
-            {% endfor %}
-            </ul>
-
-            <h3>Bank account information is</h3>
-
-            <p>
-              <b>Account holder</b>: {{ settings.ACCOUNT_NAME }}<br />
-              ACCOUNT_IBAN : {{ settings.ACCOUNT_IBAN }}<br />
-              ACCOUNT_BIC : {{ settings.ACCOUNT_BIC }}<br />
-              ref : {{ user.reference_number }}
-            </p>
-
-            <p>Best regards {{ settings.SITENAME }} hackers!</p>
-          </td>
-        </tr>
-      </table>
-    </center>
-  </body>
-</html>
+{% extends 'mail/email_base.html' %}
+
+{% load i18n %}
+
+{% block content %}
+<h1>Welcome {{user.first_name}} to {{ settings.SITENAME }}</h1>
+<p>
+  Great to have you onboard! Your membership application has been
+  approved and now is a great time to to join our community by
+  coming to the next community meeting on the first Tuesday of the
+  month or poping by weekly open house every Tuesday. It is also
+  highly recommendable to join our communication channels (Matrix,
+  Telegram or IRC and email list). Be happy to ask if you like to
+  know more!
+</p>
+
+<h2>Payment info</h2>
+<p>
+  Your personal bank reference number is
+  <b>{{user.reference_number}}</b>. Remember always to use your
+  reference number.
```

Comment:
Still here.

Reply:
doh

Reply:
actually that whole paragraph is misleading now